tests/app/dao/test_service_letter_contact_dao.py

```diff
@@ -0,0 +1,202 @@
+import pytest
+
+from app.dao.service_letter_contact_dao import (
+    add_letter_contact_for_service,
+    create_or_update_letter_contact,
+    dao_get_letter_contacts_by_service_id,
+    update_letter_contact
+)
+from app.errors import InvalidRequest
+from app.models import ServiceLetterContact
+from tests.app.db import create_letter_contact, create_service
+
+
+def test_dao_get_letter_contacts_by_service_id(notify_db_session):
```

Comment:
Do we assume service_id is always valid? Is it worth a test with an invalid service_id?

Reply:
Validation for a correct service id happens in the endpoint. Not sure there'd be much value adding it here.